com.unity.render-pipelines.high-definition/Editor/AssetProcessors/AutodeskInteractiveMaterialImport.cs

```diff
@@ -19,6 +20,9 @@ public override int GetPostprocessOrder()
 
         public void OnPreprocessMaterialDescription(MaterialDescription description, Material material, AnimationClip[] clips)
         {
+            if (GraphicsSettings.currentRenderPipeline.GetType() != typeof(HDRenderPipelineAsset))
```

Comment:
This is a null reference when no render pipeline is assigned, defeating the purpose of the PR.